lab-beeksma/lib/greet.js

```diff
@@ -0,0 +1,3 @@
+exports.greet = (name) => {
+  return typeof(name) === 'string' ? `Hello ${name}` : null;
```

Comment:
Note that `typeof` is an operator (with rather high [precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)) rather than a function, so parentheses are rarely necessary.